src/interface/interface.cpp

```diff
@@ -145,7 +145,7 @@ bool loadInterfaceResources()
 	status_bmp = loadImage("images/system/StatusBar.png");
 	character_bmp = loadImage("images/system/CharacterSheet.png");
 	hunger_bmp = loadImage("images/system/Hunger.png");
-	minotaur_bmp = loadImage("images/system/minotaur.png");
+	minotaur_bmp = loadImage("images/sprites/minotaur.png"); // the file "images/system/minotaur.png" doesn't exist in current Data
```

Comment:
No, it's fine. I'll do it.

That resource is brand new, not available in the v2.0.4 release. It will be available in the next release. For now, to get the game to run, people will need to either create a dummy file or just load in a different file on their end -- as you have done.
Maybe I should make a note somewhere concerning new resources not yet available in the steam/gog release.